lib/utils/fsUtils.js

```diff
@@ -60,7 +72,7 @@ async function createLiquidTestYaml(templateType, handle, testContent) {
   );
   const existingFile = fs.existsSync(liquidTestPath);
   if (existingFile) {
-    console.log(
+    consola.debug(
```

Comment:
Seems like it could be useful for a user as info? 
The user might not expect a liquid testing YAML to not be overwritten when importing.

Reply:
In that case it will always be shown. By default we show info. I think once you know it will create those files, you would improve your experience reducing the terminal output (it won't add any value to you)